fvtest/porttest/omrsockTest.cpp

```diff
@@ -437,21 +474,83 @@ TEST(PortSockTest, create_IPv4_mapped_IPv6_Socket_Address)
 }
 
 /**
- * Test functions to set up a connection by using two sockets, which talk to each other.
+ * Test functions to set up a stream connection by using two sockets, which talk to each other.
  *
  * First, the server starts and listens for connections. Then, the
  * client starts and sends a request to connect to the server. The messages are
  * sent both ways, and it is checked if they were sent correctly.
  * 
- * Address families tested include IPv4 and IPv6 (if supported). Socket types
- * tested include stream and datagram.
+ * Address families tested include IPv4 and socket types tested is stream.
+ *
+ * @note Errors such as failed function calls, failure to create server and/or client, wrong 
+ * message sent/received, will be reported.
+ */
+TEST(PortSockTest, two_socket_stream_communication)
+{
+	OMRPORT_ACCESS_FROM_OMRPORT(portTestEnv->getPortLibrary());
+
+	OMRSockAddrStorage serverStreamSockAddr;
+	omrsock_socket_t serverStreamSocket = NULL;
+	uint16_t port = 4930;
+	uint8_t serverAddr[4];
+
+	// To Create a Server Socket and Address
```

Comment:
We tend to use `/* blah */` style comments, rather than these (imho better) single-line comments.

Reply:
Sounds good! I will change it.